.github/pull_request_template.md

```diff
@@ -1,5 +1,36 @@
-## What are you trying to accomplish?
+Resolves #000 <!--fill issue number-->
```

Comment:
IMO this is a volunteer project and people are not likely to fill out a verbose template.